Paws4Me/Paws4Me/Display Single Pet Details/View Controllers/PetSingleDetailsViewController.swift

```diff
@@ -25,9 +25,12 @@ class PetSingleDetailsViewController: UIViewController {
         super.viewDidLoad()
         setPlaceholderImage()
         updateUI()
-        isPetSaved()
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        isPetSaved()
```

Comment:
Alignment is our here.

Also if your `ViewModel` already has `singlePetName` why do you need to pass it in 😄

Reply:
True 👍🏻